ofl/chocolateclassicalsans/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1 @@
+"Chocolate Classical Sans" is an open source font suitable for Traditional Chinese environments. It is based on the "Source Sans" font jointly developed by Adobe and Google. It partially refers to heritage fonts, including the recommended fonts of heritage font standardization documents, and other common inherited glyphs, modify the glyphs into traditional Chinese shapes and make up some missing Chinese characters.
```

Comment:
Add the "To contribute" part, like <p>To contribute, see <a href="https://github.com/fontrepo/fontname">github.com/fontrepo/fontname</a>.</p>